### ISSUE TITLE
NullPointerException in Connection.java

### ISSUE THREAD

**Issue by Fischer96** [PhilippvK/playforia-minigolf#74]
Moin,

i've started a server, all tracks loaded fine and when i try to connect with the client i just get the following error on the client:

```Java
-= AGolf =-
Copyright (c) 2002-2012 Playforia (www.playforia.info)
Programming: Pasi Laaksonen
Graphics: Janne Matilainen

Exception in thread "Thread-8" java.lang.NullPointerException
        at com.aapeli.connection.Connection.run(Connection.java:132)
        at java.lang.Thread.run(Thread.java:748)
```

And on the server:

```java
Nov 06, 2020 12:09:04 AM org.moparforia.shared.tracks.filesystem.FileSystemTrackManager load
INFO: Loaded 2055 tracks
Nov 06, 2020 12:09:04 AM org.moparforia.shared.tracks.filesystem.FileSystemTrackManager load
INFO: Loaded 7 track sets
Loaded 4 packet handler type(s)
Started server on host 0.0.0.0 with port 4242
Client connected: [id: 0xfc44fb5c, /IP => /IP]
>>> Packet{channel=[id: 0xfc44fb5c, /IP => /IP], type=COMMAND, count=0, message='new'}
Player connected: 0, reconnecting:false
>>> Packet{channel=[id: 0xfc44fb5c, /IP => /IP], type=DATA, count=0, message='version\t35'}
>>> Packet{channel=[id: 0xfc44fb5c, /IP => /IP], type=STRING, count=0, message='tlog\t1\tclientconnect\tloadtime:i:690^loadertime:i:1405'}
Client disconnected: [id: 0xfc44fb5c, /IP :> /IP]
```

Couldn't really figure out what the problem is, any ideas?

**Comment by PhilippvK** [comment]
Maybe yesterday's "Non-Functional" changes actually introduced some network-bug. I will give it a try and probably revert that commit temporarily to fix it! Thanks for your feedback.

**Comment by PhilippvK** [comment]
@Fischer96 Wow, catching that typo actually took me almost one hour.

Here is the fix: 8436611e034b39a97f8ab40bb2d28293785487db

Have fun with the game!

**Comment by Fischer96** [comment]
@PhilippvK Thanks for the quick fix, everything works fine now!
